packages/json-schemas/schemas/order_schema.json

```
@@ -1,21 +1,54 @@
 {
     "id": "/orderSchema",
     "properties": {
-        "makerAddress": { "$ref": "/addressSchema" },
-        "takerAddress": { "$ref": "/addressSchema" },
-        "makerFee": { "$ref": "/wholeNumberSchema" },
-        "takerFee": { "$ref": "/wholeNumberSchema" },
-        "senderAddress": { "$ref": "/addressSchema" },
-        "makerAssetAmount": { "$ref": "/wholeNumberSchema" },
-        "takerAssetAmount": { "$ref": "/wholeNumberSchema" },
-        "makerAssetData": { "$ref": "/hexSchema" },
-        "takerAssetData": { "$ref": "/hexSchema" },
-        "makerFeeAssetData": { "$ref": "/hexSchema" },
-        "takerFeeAssetData": { "$ref": "/hexSchema" },
-        "salt": { "$ref": "/wholeNumberSchema" },
-        "feeRecipientAddress": { "$ref": "/addressSchema" },
-        "expirationTimeSeconds": { "$ref": "/wholeNumberSchema" },
-        "domain": { "$ref": "/eip712DomainSchema" }
+        "makerAddress": {
+            "$ref": "/addressSchema"
+        },
+        "takerAddress": {
+            "$ref": "/addressSchema"
+        },
+        "makerFee": {
+            "$ref": "/wholeNumberSchema"
+        },
+        "takerFee": {
+            "$ref": "/wholeNumberSchema"
+        },
+        "senderAddress": {
+            "$ref": "/addressSchema"
+        },
+        "makerAssetAmount": {
+            "$ref": "/wholeNumberSchema"
+        },
+        "takerAssetAmount": {
+            "$ref": "/wholeNumberSchema"
+        },
+        "makerAssetData": {
+            "$ref": "/hexSchema"
+        },
+        "takerAssetData": {
+            "$ref": "/hexSchema"
+        },
+        "makerFeeAssetData": {
+            "$ref": "/hexSchema"
+        },
+        "takerFeeAssetData": {
+            "$ref": "/hexSchema"
+        },
+        "salt": {
+            "$ref": "/wholeNumberSchema"
+        },
+        "feeRecipientAddress": {
+            "$ref": "/addressSchema"
+        },
+        "expirationTimeSeconds": {
+            "$ref": "/wholeNumberSchema"
+        },
+        "chainId": {
+            "type": "number "
```

When running this through the Python tooling: `UNEXPECTED EXCEPTION: <SchemaError: "'number ' is not valid under any of the given schemas">`
```suggestion
            "type": "number"
```